wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests.m

```diff
@@ -273,14 +273,16 @@ - (void)testThatSpecialKeysAreNotPartOfTheLocallyModifiedKeys
 {
     // given
     NSSet *expected = [NSSet setWithArray:@[
-                          ZMConversationUserDefinedNameKey,
-                          ZMConversationIsForcedReadOnlyKey,
-                          ZMConversationLastReadServerTimeStampKey,
-                          ZMConversationClearedTimeStampKey,
-                          ZMConversationSilencedChangedTimeStampKey,
-                          ZMConversationArchivedChangedTimeStampKey
-                          ]];
-    
+        ZMConversationUserDefinedNameKey,
+        ZMConversationIsForcedReadOnlyKey,
+        ZMConversationLastReadServerTimeStampKey,
+        ZMConversationClearedTimeStampKey,
+        ZMConversationSilencedChangedTimeStampKey,
+        ZMConversationArchivedChangedTimeStampKey,
+        ZMConversationCipherSuite,
+        ZMConversationEpochTimestamp
```

Comment:
I don't remember the exact details but changes on tracked keys trigger some observer logic. We don't need that for the ciphersuite and the epoch timestamp. 
Take a look at `ZMConversation.swift:303` > `ignoredKeys` and add the new keys there in the same way as `ZMConversation.epochKey`. Then we can also remove the two keys from this `expected` set

Reply:
Thanks!